src/styles/WishlistScreen/listItems.jsx

```diff
@@ -0,0 +1,26 @@
+import { StyleSheet, StatusBar } from "react-native";
+
+export const sectionList = StyleSheet.create({
+    container: {
+      flex: 1,
+      paddingTop: StatusBar.currentHeight,
+      marginHorizontal: 16,
+    },
+    item: {
+      backgroundColor: '#f9c2ff',
+      padding: 20,
+      marginVertical: 8,
+    },
+    header: {
+      fontSize: 32,
+      backgroundColor: '#fff',
```

Comment:
Please add this color to your library colors

Reply:
Done.